app/controllers/sign_in_security_check_failed_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class SignInSecurityCheckFailedController < ApplicationController
+  def show
+    analytics.sign_in_security_check_failed_visited
```

Comment:
Can we add a test for this controller, to assert that it logs the expected event?